From: Nick Schermer <noreply@xfce.org>
Subject: [Xfce4-commits] <xfce4-panel:xfce-4.8> Migrate: Fix 4.6
	transparency migration (bug #7523).

Updating branch refs/heads/xfce-4.8
         to 58024ad9b4260000dd375ad68cd0cea792e86185 (commit)
       from 36173f070c5f7911d663b6ec5efdeff869c0e5eb (commit)

commit 58024ad9b4260000dd375ad68cd0cea792e86185
Author: Yves-Alexis Perez <corsac@debian.org>
Date:   Thu Apr 21 16:50:16 2011 +0200

    Migrate: Fix 4.6 transparency migration (bug #7523).
    
    (cherry picked from commit 3603b4f159e3b2ef8f9c31efd8eb5bd3e895991)

 migrate/migrate-46.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/migrate/migrate-46.c b/migrate/migrate-46.c
index fb2bece..9674be4 100644
--- a/migrate/migrate-46.c
+++ b/migrate/migrate-46.c
@@ -849,11 +849,11 @@ migrate_46_end_element_handler (GMarkupParseContext  *context,
 
           /* set transparency */
           g_snprintf (prop, sizeof (prop), "/panels/panel-%u/leave-opacity", parser->panel_id_counter);
-          xfconf_channel_set_uint (parser->channel, prop,  parser->panel_transparency);
+          xfconf_channel_set_uint (parser->channel, prop, 100 - parser->panel_transparency);
 
           g_snprintf (prop, sizeof (prop), "/panels/panel-%u/enter-opacity", parser->panel_id_counter);
           xfconf_channel_set_uint (parser->channel, prop,  parser->panel_activetrans ?
-                                   parser->panel_transparency : 100);
+                                   100 - parser->panel_transparency : 100);
 
           /* prepare for the next panel */
           parser->panel_id_counter++;
_______________________________________________
Xfce4-commits mailing list
Xfce4-commits@xfce.org
http://foo-projects.org/mailman/listinfo/xfce4-commits
